homeassistant/components/sensibo/strings.json

```diff
@@ -68,5 +68,20 @@
         }
       }
     }
+  },
+  "state_attributes": {
+    "_": {
```

Comment:
This seems incorrect, this format is used by entity components, which `Sensibo` isn't.

See: <https://developers.home-assistant.io/docs/internationalization/core#entities>

Reply:
These are for the non-climate entities attributes and works perfectly in frontend for the names but not the states.
I took a note of the info that it says it only works for climate entities but for name it works fine for all entities: https://developers.home-assistant.io/docs/internationalization/core/#entity-attribute-name-and-state-of-entity-components

Reply:
> These are for the non-climate entities

Sure, still, this is not an entity component and the format use here is not correct.

Reply:
Sorry but don't get it. It says it's for translating attributes for entities?
Didn't intend to translate states for entity components by this one.
Seems to me `state_attributes` is a bit double purpose?

Reply:
> Didn't intend to translate states for entity components by this one.

That you are using the wrong chapter from the developer docs. 

<https://developers.home-assistant.io/docs/internationalization/core/#state-of-entities>

an example in our codebase can be found here:

<https://github.com/home-assistant/core/blob/9ed629d838ba4bbff87e8de9870a3d8875eb49a6/homeassistant/components/plugwise/strings.json#L28-L43>